### ISSUE TITLE
not working?

### ISSUE THREAD

**Issue by Anonyfox** [minibikini/paasaa#9]
```sh
iex(1)> Paasaa.all "hello world"
** (Protocol.UndefinedError) protocol Enumerable not implemented for nil. This protocol is implemented for: Ecto.Adapters.SQL.Stream, Postgrex.Stream, Floki.HTMLTree, DBConnection.Stream, DBConnection.PrepareStream, Timex.Interval, HashSet, Range, Map, Function, List, Stream, Date.Range, HashDict, GenEvent.Stream, MapSet, File.Stream, IO.Stream
    (elixir) /private/tmp/elixir-20190202-25152-hzjos7/elixir-1.8.0/lib/elixir/lib/enum.ex:1: Enumerable.impl_for!/1
    (elixir) /private/tmp/elixir-20190202-25152-hzjos7/elixir-1.8.0/lib/elixir/lib/enum.ex:141: Enumerable.reduce/3
    (elixir) lib/enum.ex:2486: Enum.take/2
    (elixir) lib/stream/reducers.ex:31: anonymous fn/3 in Stream.Reducers.chunk_every/5
    (elixir) lib/enum.ex:504: Enum.chunk_while/4
    (paasaa) lib/paasaa.ex:228: Paasaa.n_grams/2
    (paasaa) lib/paasaa.ex:203: Paasaa.get_clean_trigrams/1
    (paasaa) lib/paasaa.ex:126: Paasaa.process/2
```

tried with 

```sh
$ elixir -v
Erlang/OTP 21 [erts-10.2.3] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.8.0 (compiled with Erlang/OTP 21)
```

**Comment by minibikini** [comment]
Thanks for reporting, @Anonyfox!

I just fixed it, please update `paasaa` to 0.3.1 in your `mix.exs`.

**Comment by Anonyfox** [comment]
it works, thank you!
